### DISCUSSION TITLE
Workflow for one database, two computers

### DISCUSSION THREAD

**Discussion by piramiday** [keepassxreboot/keepassxc#9135] [Q&A] [answered]
Hello, I would like to have a better workflow in working with one database but with two different browsers.

Ideally, I would like to manually change the KeepassXC-Browser settings for my browsers so that the IDs check out and I can move my kdbx file from one computer to the other one without severing the connection between each browser and the database.
At present, instead, I have computer A with browser 1 and 2 with keys A1 and A2, and computer B with browser 1 and 2 with keys B1 and B2.
If I update a password on computer A, I would like to be able to easily copy the kdbx file over computer B without losing track of the browser connections.

Does anyone have a proper solution for this? Thanks!

**Comment by droidmonkey** [comment, marked as answer]
As long as you are using unique names for your browser connections then this should just work.

**Comment by piramiday** [comment]
I'm sorry, I'm not sure I understand: unique names means the name that I type as an identifier for the browser extension? or its ID hash string?
I thought I would have got to a point where I had only a name/id string and then I could replicate it to the second computer.
Are you thinking about the opposite, that is, have a DB with two connections for the two computer browsers, only one of which will be actually expressed at each computer?

**Reply by droidmonkey** [reply]
For each browser on each computer you need to set a unique _name_ for the connection for each database. If you don't, then you will overwrite another browser's connection and when you sync the database then it will fail to connect. So in your example you will have _four_ defined connections in your database. 

![image](https://user-images.githubusercontent.com/2809491/219953378-be432d3d-2e8e-4b25-b736-3bf81752debd.png)
